skyplane/cli/cli.py

```diff
@@ -12,6 +12,7 @@
 import skyplane.cli.usage.definitions
 import skyplane.cli.usage.client
 from skyplane.cli.usage.client import UsageClient, UsageStatsStatus
+from skyplane.obj_store.azure_storage_account_interface import AzureStorageAccountInterface
```

Comment:
Why do we need this import?

Reply:
my bad, it was an artifact from debugging. I'll remove